src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  background-color: $white;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $grey;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 40px 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $black;
+  }
+
+  &__code {
+    font-family: $roboto;
```

Comment:
Let's follow DRY(Dont repeat yourself)
You already declare the font-family for all HTML

Reply:
done